README.md

```diff
@@ -55,25 +61,65 @@ spec:
   restartPolicy: Always
   ```
 
-## Installing a different Node.js version
+### Installing a Different Node.js Version
 
 The image includes the `install_node` script that can be used to install a particular Node.js version. For example:
 
-        FROM launcher.gcr.io/google/nodejs
+```docker
+FROM launcher.gcr.io/google/nodejs
 
-        # Install node.js 4.6.1
-        RUN install_node v4.6.1
+# Install node.js 4.6.1
+RUN install_node v4.6.1
 
-        # Copy application code.
-        COPY . /app/
+# Copy application code.
+COPY . /app/
 
-        # Install dependencies.
-        RUN npm --unsafe-perm install
+# Install dependencies.
+RUN npm --unsafe-perm install
+```
 
 Node.js is installed with binary packages hosted on a Google-provided mirror.
 
+## The Node.js Runtime Builder
```

Comment:
My opinion is that up-to-date runtime details should absolutely be documented at https://cloud.google.com/appengine/docs/flexible/nodejs/runtime